### DISCUSSION TITLE
What is `@milkdown/exception`?

### DISCUSSION THREAD

**Discussion by zgrybus** [Milkdown/milkdown#1047] [General]
I don't see any information about this plugin. What is it doing?
Does it inform us, that there were a parsing error? 🤔 
CC: @Saul-Mirone

**Comment by Saul-Mirone** [comment]
It's a package to provide common error and error messages for official packages in milkdown. The errors has a lot of kinds and some of them are parsing error. They all carry some error messages and codes.

**Reply by zgrybus** [reply]
@Saul-Mirone Is there any API for that?

**Reply by Saul-Mirone** [reply]
They are just error instances so you can check the error.code or use `instanceof` to check it.
